docs/source/shared/add-plugin.mdx

```diff
@@ -1,21 +1,21 @@
 import {MultiCodeBlock} from 'gatsby-theme-apollo-docs';
 
-Get the latest version of Apollo Android from [the Releases page](https://github.com/apollographql/apollo-android/releases).
+First, obtain the latest version number of Apollo Android from [the Releases page](https://github.com/apollographql/apollo-android/releases).
 
-In your app Gradle file, apply the `com.apollographql.apollo3` plugin.
+Then in your app's Gradle file, apply the `com.apollographql.apollo3` plugin using one of the following methods (replace `x.y.z` with the desired version number):
```

Comment:
I sometimes use `$version` instead of `x.y.z`. Might be worth uniformizing the pattern? I tend to like `$version` better as it will look more familiar to a Kotlin reader expecting a string template. Any thoughts?

Reply:
I agree we should standardize. If we go with `$version`, will that confuse any _other_ portion of our readers?

In most other docsets I'd use something large and noticeable, like "PLUGIN_VERSION_NUMBER". Would that work here or cause its own confusion?

Reply:
`$version` works well for Kotlin and a bunch of other languages. I'm thinking of shell where that'd be a variable substitution as well. That doesn't work in Java though so maybe "VERSION_NUMBER" is better. Since there's only one version and it applies to the plugin + runtime + api + cache + ... I'd drop the "PLUGIN_" prefix.